notes-with-indexeddb/background.js

```diff
@@ -0,0 +1,14 @@
+/*
+Open a new tab, and load "notes.html" into it.
+*/
+function openNotes() {
```

Comment:
It would be nice to query whether a matching tab already exists and switch to it if it does, but I'm happy to leave that for a follow-up.